apps/asap-server/validate-orcids-script/validate-orcids.ts

```diff
@@ -0,0 +1,28 @@
+// This is a one-off script. It was run on prod on <date> and the results (invalid ORCIDs) were passed on to product
+import Users from '../src/controllers/users';
+
+import { VALID_ORCID } from '@asap-hub/validation';
+
+const validateOrcids = async (): Promise<void> => {
+  const users = new Users();
+
+  const maxNumberOfUsers = 999999;
+  const allUsers = (await users.fetch({ take: maxNumberOfUsers })).items;
+
+  let noInvalidOrcids = true;
+
+  for (const user of allUsers) {
+    if (user.orcid && user.orcid?.match(VALID_ORCID) === null) {
+      noInvalidOrcids = false;
+      // eslint-disable-next-line no-console
+      console.log(`Invalid orcid on user ${user.id}: ${user.orcid}`);
+    }
+  }
+
+  if (noInvalidOrcids) {
+    // eslint-disable-next-line no-console
+    console.log('No users with invalid orcids found.');
+  }
+};
+
+validateOrcids();
```

Comment:
This script will be run by ssh'ing into prod then `TS_NODE_COMPILER_OPTIONS='{"module": "commonjs"}' yarn ts-node apps/asap-server/validate-orcids-script/validate-orcids.ts`

Reply:
is there any point of adding this to codebase? i assume this is a one off to make sure all orcids are valid - if not what is the action?

Reply:
eh, may as well keep the knowledge in there, may help in the future

Reply:
cool, then maybe you should consider adding some sort of README next to it or something and provide a command to run it as well some description?

Reply:
@ana-yldio maybe add the link about how to run this script too? its here in this comment thread